music21/stream/__init__.py

```diff
@@ -11024,8 +11012,6 @@ def activateVariants(self, group=None, *, matchBySpan=True, inPlace=False):
         # Now fix measure numbers given the saved information
         returnObj._fixMeasureNumbers(deletedMeasures, insertedMeasures)
 
-        # have to clear cached variants, as they are no longer the same
-        returnObj.coreElementsChanged()
```

Comment:
If you don't mind, leave this, unless you're absolutely clear on the variant code -- it's old code and not super well documented or tested, so there might be some core-leaking there.